.gitignore

```diff
@@ -17,7 +17,7 @@ node_modules
 coverage/
 package-lock.json
 yarn-error.log
-.storybook-static
+storybook-static
```

Comment:
This change is resulting in a lot of tracked files generated by `build-storybook` command. Is it on purpose?

Reply:
@clxandstuff oh it might be that I see previously generated files being tracked after I switched branches, no issue then